### PR TITLE
fix: Make new meeting dialog full screen on mobile

### LINKED ISSUE #6977: Make new meeting dialog full screen on mobile
Make new meeting dialog full screen on mobile.

image

AC: 
- full screen on mobile
- keep modal on desktop

### PR DESCRIPTION
**Pull request by gitstart**
# Description

Fixes #6977
 
image
 
 AC:
 
 * full screen on mobile
 * keep modal on desktop

## Demo

[If possible, please include a screenshot or gif/video, it'll make it easier for reviewers to understand the scope of the changes and how the change is supposed to work. If you're introducing something new or changing the existing patterns, please share a Loom and explain what decisions you've made and under what circumstances]

## Testing scenarios

[Please list all the testing scenarios a reviewer has to check before approving the PR]

- [ ] Scenario A
  - Step 1
  - Step 2...

- [ ] Scenario B
  - Step 1
  - Step 2....

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [x] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/client/components/NewMeetingActions.tsx [hunk 1]
```diff
@@ -27,7 +27,11 @@ const ActionRow = styled('div')({
   padding: 24,
   paddingTop: 16,
   [narrowScreenMediaQuery]: {
-    flexDirection: 'column'
+      flexDirection: 'column',
```
Comment (igorlesnenko) [inline packages/client/components/NewMeetingActions.tsx:30]: Can we stick buttons to the bottom of the screen on mobile?
Probably something like:

```
 margin-top: auto;
 padding-bottom: 24px;
}
```

**Review by igorlesnenko — CHANGES_REQUESTED** [review]
Thank you for the PR! Only one little change:

File: packages/client/components/NewMeetingActions.tsx (hunk 1 above)
Comment (gitstart) [reply packages/client/components/NewMeetingActions.tsx:30]: Thanks for the review, would work on that

**Review by igorlesnenko — APPROVED** [review]
Thank you! Looks good to me!

**Pull request merged**